random.c

```diff
@@ -1644,8 +1659,32 @@ rb_f_rand(int argc, VALUE *argv, VALUE obj)
  * call-seq:
  *   Random.rand -> float
  *   Random.rand(max) -> number
+ *
+ * When +max+ is an Integer, +rand+ returns a random integer greater than
+ * or equal to zero and less than +max+. Unlike Kernel.rand, when +max+
+ * is a negative integer or zero, +rand+ raises an ArgumentError.
+ *
+ *   Random.rand(100)       # => 42
+ *
+ * When +max+ is a Float, +rand+ returns a random floating point number
+ * between 0.0 and +max+, including 0.0 and excluding +max+.
+ *
+ *   Random.rand(1.5)       # => 1.4600282860034115
+ *
+ * When +max+ is a Range, +rand+ returns a random number where
+ * range.member?(number) == true.
+ *
+ *   Random.rand(5..9)      # => one of [5, 6, 7, 8, 9]
+ *   Random.rand(5...9)     # => one of [5, 6, 7, 8]
+ *   Random.rand(5.0..9.0)  # => between 5.0 and 9.0, including 9.0
+ *   Random.rand(5.0...9.0) # => between 5.0 and 9.0, excluding 9.0
+ *
+ * Both the beginning and ending values of the range must respond to subtract
+ * (<tt>-</tt>) and add (<tt>+</tt>)methods, or rand will raise an
+ * ArgumentError.
+ *
```

Comment:
Duplication of `Random#rand`, and users would have to read carefully to know what differs.
```suggestion
 * Returns a random number using the Ruby system PRNG.
 *
 * See also Random#rand.
```